src/Elastic.Apm/DiagnosticListeners/HttpDiagnosticListener.cs

```diff
@@ -1,133 +1,34 @@
-﻿using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.Linq;
-using System.Net.Http;
-using System.Reflection;
-using Elastic.Apm.Api;
-using Elastic.Apm.Config;
+﻿using System.Runtime.InteropServices;
 using Elastic.Apm.DiagnosticSource;
-using Elastic.Apm.DistributedTracing;
 using Elastic.Apm.Helpers;
 using Elastic.Apm.Logging;
-using Elastic.Apm.Model;
 
 namespace Elastic.Apm.DiagnosticListeners
 {
 	/// <inheritdoc />
```

Comment:
nit: the `<inheritdoc>` became obsolete, because there is no interface which is implemented by this. It can be removed.

Reply:
Done.